apps/02RockPaperScissors.js

```diff
@@ -7,7 +7,58 @@ prompt.start();
 
 function rockPaperScissors(hand1, hand2) {
 
-    // Write code here
+    // // Write code here
+    var options = ["rock", "scissors", "paper"];
+    var compute_option = Math.floor( Math.random() * 3 );
+    // other option, if (Math.random() < .5) ? "Option1" "Option2";
+
+    hand1 = hand1.toLowerCase();
+    hand2 = hand2.toLowerCase();
+
+    if ( !(options.indexOf(hand1) > -1) ) {
+      return "Hand One, Please Enter a correct value: rock, scissors or paper";
```

Comment:
Nice input sanitization! We could probably simplify this conditional... can you think of how?